feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -430,13 +430,24 @@ fun ZoomScaleText(zoomScale: Float) {
     )
     Text(
         modifier = Modifier
+            .padding(horizontal = 10.dp)
             .alpha(contentAlpha.value)
             .testTag(ZOOM_RATIO_TAG),
         text = "%.1fx".format(zoomScale),
         fontSize = 20.sp
     )
 }
 
+@Composable
+fun CurrentCameraIdText(physicalCameraId: String?, logicalCameraId: String?) {
+    Text(
+        modifier = Modifier
+            .testTag(CURRENT_CAMERA_ID_TAG),
+        text = "physical id: $physicalCameraId, logical id: $logicalCameraId",
```

Comment:
To make this easier to parse by UI automator, we should split it up so that the physical and logical IDs have their own tag. Something like this:
```
@Composable
fun CurrentCameraIdText(physicalCameraId: String?, logicalCameraId: String?) {
    Column(horizontalAlignment = Alignment.CenterHorizontally) {
        Row {
            Text(text = "Logical ID: ")
            Text(
                modifier = Modifier.testTag(LOGICAL_CAMERA_ID_TAG),
                text = logicalCameraId ?: "---"
            )
        }
        Row {
            Text(text = "Physical ID: ")
            Text(
                modifier = Modifier.testTag(PHYSICAL_CAMERA_ID_TAG),
                text = physicalCameraId ?: "---"
            )
        }
    }
}
```